content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -1332,4 +1330,3 @@ guide is available
 [here](/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/). We are
 also making investments in cluster API to ensure Windows nodes are properly
 provisioned.
-
```

Comment:
surprise it wasn't flagged for not having a blank line at end...

Reply:
Not sure how this got dropped but I added it back.